accounts/templates/accounts/edit_profile.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1>Edit Profile</h1>
+<div class="container">
```

Comment:
The base template already has an element with a `container` class, so I would remove this div.